Google.Cloud.Compute.V1.AcceptanceTests/MainIntegrationTest.cs

```diff
@@ -24,22 +24,43 @@ public sealed class MainIntegrationTest
         private readonly AddressesClient _addrClient;
         private readonly RegionOperationsClient _ropsClient;
 
-        const string CredentialsPath = @"C:\Users\virost\AppData\Roaming\gcloud\legacy_credentials\virost@google.com\adc.json";
+        private const string TestCredentialsEnvironmentVariable = "GOOGLE_APPLICATION_CREDENTIALS";
+        private const string TestProjectEnvironmentVariable = "COMPUTE_TEST_PROJECT";
```

Comment:
(No change needed). We use a different environment variable (TEST_PROJECT) in google-cloud-dotnet; this will change a little when I move the tests over.